scriptworker/cot/verify.py

```diff
@@ -1138,9 +1138,13 @@ def verify_decision_command(decision_link, mach_commands=DECISION_MACH_COMMANDS)
     pushlog_id = list(pushlog_info['pushes'].keys())[0]
     decision_comment = get_commit_message(decision_link.task)
     push_comment = pushlog_info['pushes'][pushlog_id]['changesets'][0]['desc']
+    allowed_comments = [' ']
     # try syntax uses the first line of the commit.
     first_line = push_comment.split('\n')[0]
-    if decision_comment not in (' ', first_line):
+    if 'try:' in first_line:
+        parts = first_line.split('try:')
```

Comment:
Maybe something like:
```python
_, try_syntax = first_line.split('try:', 1)
```
?

It [looks](https://github.com/taskcluster/mozilla-taskcluster/blob/cf5ce67127c694e01a9fa3f0011e9aae6ac5169e/src/jobs/taskcluster_graph.js#L43-L58) like taskcluster splits at the first occurrence of `try:` in a line. Actually, it looks like it will find that syntax anywhere in the commit message even, though I'm not sure we want to care.